### PR TITLE
add networkpolicy wkld

### PR DESCRIPTION
**Pull request by mohit-sheth**
### Description
Check out networkpolicy/README below
Cases 1 and 4 are not in this PR's scope

TODO
- [x] Viz for results

### CODE REVIEW

File: workloads/kube-burner/workloads/networkpolicy/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## How to run networkpolicy tests?
+
+The environmental variables and steps to kick off this test can be found [here](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner/README.md#kube-burner-e2e-benchmarks)
+
+## What are the test cases that this workload can currently run?
+
+A networkpolicy can come in various shapes and sizes. Allow traffic from a specific namespace, Deny traffic from a specific pod IP, Deny all traffic, ...
+Hence we have come up with a few test cases which try to cover most of them. They are as follows.
+
+### Case 2
```
Comment (smalleni) [inline workloads/kube-burner/workloads/networkpolicy/README.md:15]: Perhaps add all cases to the list and explain them and then call out the ones specifically implemented - so it makes sense when you say case2 and case3?

**Review by smalleni — APPROVED** [review]
lgtm

File: workloads/kube-burner/workloads/networkpolicy/clusterrole.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list-1
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
```
Comment (afcollins) [inline workloads/kube-burner/workloads/networkpolicy/clusterrole.yml:8]: This is equivalent to cluster-admin.

File: workloads/kube-burner/workloads/networkpolicy/clusterrolebinding.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list-crb-1
+subjects:
+- kind: Group
+  name: system:authenticated
```
Comment (afcollins) [inline workloads/kube-burner/workloads/networkpolicy/clusterrolebinding.yml:7]: .. granted to all users.

Comment (afcollins) [reply workloads/kube-burner/workloads/networkpolicy/clusterrolebinding.yml:7]: If this role binding is for pod-scraper, it looks like only view perms are required on pods and networkpolicies inside the namespace.

Not a requirement, just a nit

**Review by afcollins — APPROVED** [review]
LGTM

File: workloads/kube-burner/workloads/networkpolicy/case3.yml [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+          label2: "bar"
+          workload: ${WORKLOAD}
+          es_server: ${ES_SERVER}
+          es_index: ${ES_INDEX_NETPOL}
+          set: "4"
+
+      - objectTemplate: nginx.yml
+        replicas: 5
+        inputVars:
+          nodeSelector: "${POD_NODE_SELECTOR}"
```
Comment (afcollins) [inline workloads/kube-burner/workloads/networkpolicy/case3.yml:140]: nice!

**Pull request merged**